### DISCUSSION TITLE
Slow Initial Page Load Time Despite Cache Clearing and Setting in Filament Demo

### DISCUSSION THREAD

**Discussion by dpenacton** [filamentphp/filament#12752] [Help]
### Package

filament/filament

### Package Version

3.0.0

### Laravel Version

v11.0

### Livewire Version

3.0.0

### PHP Version

8.3.0

### Problem description

When attempting to clear and set the cache for the Filament demo on GitHub, there persists a significant delay in loading the initial page, taking over 40 seconds. Despite cache clearing and reconfiguration, the issue persists, impacting the user experience.

Despite the cache clearing and reconfiguration process, the initial page load time remains slow, taking over 40 seconds. Subsequent page loads show improved performance, indicating a potential caching issue during the initial load.

### Expected behavior

It should load faster once all the caches are set

### Steps to reproduce

Steps to Reproduce:
1. Clone the Filament demo repository: `git clone https://github.com/laravel-filament/demo.git filament-demo && cd filament-demo`
2. Install dependencies: `composer install --ignore-platform-req=ext-pcntl --ignore-platform-req=ext-posix`
3. Copy the environment file: `cp .env.example .env`
4. Generate a Laravel application key: `php artisan key:generate`
5. Migrate the database: `php artisan migrate`
6. Seed the database: `php artisan db:seed`
7. Create a symbolic link to storage: `php artisan storage:link`
8. Clear and set the cache:
   ```
   php artisan o:c
   php artisan view:cache
   php artisan route:cache
   php artisan icons:cache
   php artisan filament:cache-components
   php artisan event:cache
   php artisan config:cache
   php artisan optimize
   ```

### Reproduction repository

https://github.com/filamentphp/demo

**Comment by danharrin** [comment]
Converting this to a discussion, as I use this repo every day and if I could reproduce the issue then I would have noticed it by now

**Comment by pionl** [comment]
Hi, I've the same issue on first render. Currently made just 1 panel with 1 resource (list of users), 2 relation managers to the resource.

The first load is very slow. I've checked sentry and it seems that some of the views are not cached and the rendering takes a very long time.

I'm happy to share a screen of code, sentry logs, etc. It is very strange. Running in kubernetes on DO (nvme disk).

<img width="1649" alt="Snímek obrazovky 2024-07-21 v 23 20 59" src="https://github.com/user-attachments/assets/cd218430-a615-4e1f-8c2b-4f364d67fa67">

OPCache is set:

```
/app $ php -i | grep opcache
/usr/local/etc/php/conf.d/docker-php-ext-opcache.ini,
opcache.blacklist_filename => no value => no value
opcache.consistency_checks => 0 => 0
opcache.dups_fix => Off => Off
opcache.enable => On => On
opcache.enable_cli => On => On
opcache.enable_file_override => Off => Off
opcache.error_log => no value => no value
opcache.file_cache => no value => no value
opcache.file_cache_consistency_checks => On => On
opcache.file_cache_only => Off => Off
opcache.file_update_protection => 2 => 2
opcache.force_restart_timeout => 180 => 180
opcache.huge_code_pages => Off => Off
opcache.interned_strings_buffer => 16 => 16
opcache.jit => tracing => tracing
opcache.jit_bisect_limit => 0 => 0
opcache.jit_blacklist_root_trace => 16 => 16
opcache.jit_blacklist_side_trace => 8 => 8
opcache.jit_buffer_size => 0 => 0
opcache.jit_debug => 0 => 0
opcache.jit_hot_func => 127 => 127
opcache.jit_hot_loop => 64 => 64
opcache.jit_hot_return => 8 => 8
opcache.jit_hot_side_exit => 8 => 8
opcache.jit_max_exit_counters => 8192 => 8192
opcache.jit_max_loop_unrolls => 8 => 8
opcache.jit_max_polymorphic_calls => 2 => 2
opcache.jit_max_recursive_calls => 2 => 2
opcache.jit_max_recursive_returns => 2 => 2
opcache.jit_max_root_traces => 1024 => 1024
opcache.jit_max_side_traces => 128 => 128
opcache.jit_prof_threshold => 0.005 => 0.005
opcache.lockfile_path => /tmp => /tmp
opcache.log_verbosity_level => 1 => 1
opcache.max_accelerated_files => 20000 => 20000
opcache.max_file_size => 0 => 0
opcache.max_wasted_percentage => 5 => 5
opcache.memory_consumption => 256 => 256
opcache.opt_debug_level => 0 => 0
opcache.optimization_level => 0x7FFEBFFF => 0x7FFEBFFF
opcache.preferred_memory_model => no value => no value
opcache.preload => no value => no value
opcache.preload_user => no value => no value
opcache.protect_memory => Off => Off
opcache.record_warnings => Off => Off
opcache.restrict_api => no value => no value
opcache.revalidate_freq => 0 => 0
opcache.revalidate_path => Off => Off
opcache.save_comments => On => On
opcache.use_cwd => On => On
opcache.validate_permission => Off => Off
opcache.validate_root => Off => Off
opcache.validate_timestamps => Off => Off
```

Cache is made:

<img width="588" alt="Snímek obrazovky 2024-07-21 v 23 22 50" src="https://github.com/user-attachments/assets/86599fdd-d723-4bc6-a5ad-d7b43354ba92">
